src/features/board/Board.tsx

```diff
@@ -1,12 +1,39 @@
-import React from 'react';
+import React, { useEffect } from 'react';
 import { board, attack } from './boardSlice';
+import {selectCard, unselectCard, selectSelectedCard} from '../card/selectedCardSlice';
 import { useAppSelector, useAppDispatch } from '../../app/hooks';
 import { CardInterface, Card } from '../card/Card';
+import {useSelector} from "react-redux";
```

Comment:
I dont really care much but just to inform you, when I use destructuration, I like to put space like this `{ useSelector }`

I wont block the MR if you dont do it, this is just information about how I code style this kind of stuff